appveyor.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.0.{build}
```

Comment:
Is the `version` part required, btw? Until we start making releases, I don't see the point.